content/documentation/commands.md

```diff
@@ -0,0 +1,105 @@
++++
+title = "Commands"
```

Comment:
What about `Phel Commands` or `CLI Commands`?

Reply:
I will use `CLI Commands`, thanks! 

And regarding the weight, I am not sure. I will leave it at 21, and let decide/change it to @jenshaase to another level if he wants 😄 (in this or in a follow up commit/PR)